xml/System.Runtime.CompilerServices/SwitchExpressionException.xml

```diff
@@ -63,8 +63,8 @@
         <Parameter Name="innerException" Type="System.Exception" />
       </Parameters>
       <Docs>
-        <param name="innerException">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="innerException">The exception that is the cause of the current exception. If the <paramref name="innerException" /> parameter is not a null reference, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</param>
```

Comment:
`a null reference` => `<see langword="null" />`

I wonder if the second sentence is necessary, or important for the user. Maybe it could go in the remark?

```suggestion
        <param name="innerException">The exception that is the cause of the current exception. If the <paramref name="innerException" /> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</param>
```

Reply:
There are a lot of cases in our docs where we are doing the same thing eg https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.AddIn.Hosting/InvalidPipelineStoreException.xml#L150